src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/index.js

```diff
@@ -22,33 +22,33 @@ import GordonPeopleSearch from 'components/Header/components/PeopleSearch';
 import ApplicantListItem from './components/ApplicantListItem';
 
 /**
+ * @typedef { import('services/housing').ApplicationDetails } ApplicationDetails
  * @typedef { import('services/user').StudentProfileInfo } StudentProfileInfo
  */
 
-// Create a list of applicants, displayed by name, username, and class standing.
+/**
+ * Renders the list of applicants, displayed by name, username, and class standing.
+ * @param {Object} props The React component props
+ * @param {Boolean} props.disabled Boolean to disable the interactive elements of this list item
+ * @param {*} props.authentication The user authentication
```

Comment:
`authentication` is a simple Boolean representing where the user is authenticated (I didn't name it).

Reply:
I have also been going back and forth about removing it from these subcomponents because it is impossible to even view the student or staff page without being authenticated. The top-level `ApartmentApp` component handles that, so it is redundant in it's subcomponents

Reply:
I think removing it is a totally valid option then!

One of the eventual goals is to use the React Context API to handle Auth state more robustly and with a much neater solution for deeply nested components that need to be aware of auth state.